components/bulletins/BulletinCard.vue

```diff
@@ -56,24 +56,28 @@ export default {
 
 <style lang="postcss" scoped>
 .bulletinCard {
-    @apply relative flex flex-col items-center font-serif rounded-3xl border-4;
-    width: 47%;
-    @media (min-width: 415px) {
-        width: 214px;
-    }
-    border-color: #4b4b4b;
-    color: #4b4b4b;
+    @apply relative flex flex-col justify-center items-center rounded-xl;
+    padding: 25px 3px;
+    width: 236px;
```

Comment:
The fixed padding and width may break the layout in RWD styling.

Reply:
Used min-width/height instead.